book/src/design/circuit/gadgets/ecc/fixed-base-scalar-mul.md

```diff
@@ -25,23 +23,97 @@ $$
 \end{array}
```

Comment:
Just above here there is a `\cdot 1`; what is that for?

Reply:
I think that was when we had an extra factor of `1` in the impl. I'm going to remove this `\cdot 1` since we no longer have that.